apikit/convenience.py

```diff
@@ -335,6 +423,25 @@ def __init__(self, name, version, repository, description,
                            api_version=api_version,
                            auth=auth,
                            route=route)
+        logfile = None
+        syslog = False
+        loghost = None
+        loglevel = None
+        if "LOGFILE" in os.environ and os.environ["LOGFILE"]:
```

Comment:
We still want an intermediate layer so you can override where your log goes; look at the testing routine for an example.